frontend/test/metabase/scenarios/embedding/embedding-native.cy.spec.js

```diff
@@ -35,6 +35,17 @@ describe("scenarios > embedding > native questions", () => {
       cy.contains("39.58");
 
       filterWidget().should("not.exist");
+
+      // Test embedding with locale metabase#22561
```

Comment:
I piggy bag this test here as I remember @nemanjaglumac told me to not create a new test if not necessary as there is an overhead for Cypress.

Reply:
You can use the question with id = 1 for the test (i.e. `/question/1`). There would be little overhead for creating a test then.